src/sidebar/app/utils/utils.js

```diff
@@ -11,4 +11,45 @@ function formatFooterTime(date) {
   });
 }
 
-export { formatFooterTime };
+/**
+ *
+ * @param {HTMLElement Object} parentElement
+ * @returns {HTMLElement Object}
+ */
+function getFirstNonEmptyElement(parentElement) {
+  // create an Array from parentElement's `children` (HTMLCollection)
+  let parentElementChildrenArray = [];
+  if (parentElement.children.length > 20) {
+    parentElementChildrenArray = Array.prototype.slice.call(parentElement.children[0, 20]);
+  } else {
+
```

Comment:
no need for this empty new line, but I have a feeling you won't need this `else` anyway

Reply:
Jeez, nice catch! There was a `console.log` statement there and somehow I didn't get rid of the newline :P